### ISSUE TITLE
What scheduler is used for segmentation?

### ISSUE THREAD

**Issue by JKL-uxf** [NVlabs/MambaVision#26]
First, thank you for the paper, it's really awe-inspiring.

I know you will release segmentation version of code someday after, but I can't wait for testing this model at segmentation area. 
So I want to try this model with classification pre-training weight.  But when I read your paper, I've wondered what 3x scheduler you used for segmentation scheduler.

Is it cosine decay scheduler with 3 x longer than classification scheduler's decay steps?

**Comment by ahatamiz** [comment]
Hi @JKL-uxf 

As explained [here](https://github.com/facebookresearch/detectron2/issues/119), The definition of 1x and 3x schedules are as follows:

1x = 16 images / iterations * 90,000 iterations in total with the LR reduced at 60k and 80k
3x = 16 images / iteration * 270,000 iterations in total with the LR reduced at 210k and 250k

3x scheduler MaskRCNN scheduler was first introduced in [here](https://arxiv.org/pdf/1912.08193) and subsequently used in many other works like [Swin Transformers](https://arxiv.org/pdf/2103.14030).   

We will provide segmentation/detection code/models soon.

**Comment by JKL-uxf** [comment]
@ahatamiz  Thank you for answer.

**Comment by JKL-uxf** [comment]
@ahatamiz Thank you for answer.
